### ISSUE TITLE
[TF2] CTF MEDIC UBER BUG.

### ISSUE THREAD

**Issue by MUXAMEDPVO-MeeMDi** [ValveSoftware/Source-1-Games#4609]
https://www.youtube.com/watch?v=5fUEa680xWg a video proof how it looks in the game.

**Comment by just-a-github-user** [comment]
Nah, ubering someone after they pick up the intel works on any class on any weapon, I also don't think it's game breaking at all, I mean, how the hell do you walk into your own base in 8 seconds-ish?

**Comment by MikhailaTF** [comment]
It's especially broken if it does work on any class and any weapon. Takes only an averagely skilled soldier or demoman (or just a regular scout) to escape the enemy base with the intel under 8 seconds on maps like landfall or doublecross.

Regardless this is clearly not an intended mechanic and should be fixed accordingly.
